app/models/plan.rb

```diff
@@ -218,7 +218,8 @@ def request_feedback(user)
 
         if self.save!
           # Send an email confirmation to the owners and co-owners
-          deliver_if(recipients: self.owner_and_coowners, key: 'users.feedback_requested') do |r|
+          owners = User.joins(:roles).where('roles.plan_id =? AND roles.access IN (?)', self.id, Role.access_values_for(:administrator))
```

Comment:
owner_and_coowners class method is still used after the change? If not, please remove it!